catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/dialog/login/SignInDialog.java

```diff
@@ -342,7 +342,7 @@ private void triggerGPlusLogin(GoogleSignInResult result) {
 			GoogleSignInAccount account = result.getSignInAccount();
 			onGoogleLogInComplete(account);
 		} else {
-			ToastUtil.showError(getActivity(), "ERROR Google+ Signin. Status: " + result.getStatus());
+			ToastUtil.showError(getActivity(), getString(R.string.error_google_plus_sign_in, result.getStatus().getStatusMessage()));
```

Comment:
Even though the documentation for the Status class is non existent, let's go with that.

Reply:
`result.getStatus()` will return an object of type Status
but `result.getStatus().getStatusMessage()` will return a string therefor i used it.
did i do it wrong?

Reply:
> did i do it wrong?

Quite the opposite actually! :wink: I like your solution.

I was just mentioning that there is no [documentation for the getStatusMessage method](https://developers.google.com/android/reference/com/google/android/gms/common/api/Status#getStatusMessage()). I also couldn't find more information online. There are some code snippets at https://developers.google.com/games/services/android/signin which are using `getStatusMessage` as well. However, it still would've been great to see some example messages that `getStatusMessage` return.

I can't say for sure what `result.getStatus().toString()` returns but it looks like it's returning a String only meant to be read by developers. `getStatusMessage` seems better suited. So, let's go with your solution! :1st_place_medal: